staking_deposit/settings.py

```diff
@@ -10,30 +10,30 @@ class BaseChainSetting(NamedTuple):
 
 
 MAINNET = 'mainnet'
+ROPSTEN = 'ropsten'
+GOERLI = 'goerli'
 PRATER = 'prater'
-KINTSUGI = 'kintsugi'
 KILN = 'kiln'
-ROPSTEN = 'ropsten'
 
 
 # Mainnet setting
 MainnetSetting = BaseChainSetting(NETWORK_NAME=MAINNET, GENESIS_FORK_VERSION=bytes.fromhex('00000000'))
 # Ropsten setting
 RopstenSetting = BaseChainSetting(NETWORK_NAME=ROPSTEN, GENESIS_FORK_VERSION=bytes.fromhex('80000069'))
-# Testnet (spec v1.0.1)
-PraterSetting = BaseChainSetting(NETWORK_NAME=PRATER, GENESIS_FORK_VERSION=bytes.fromhex('00001020'))
-# Merge Testnet (spec v1.1.4)
-KintsugiSetting = BaseChainSetting(NETWORK_NAME=KINTSUGI, GENESIS_FORK_VERSION=bytes.fromhex('60000069'))
+# GOERLI (PRATER is alias)
+_GOERLI_GENESIS_FORK_VERSION = bytes.fromhex('00001020')
+GoerliSetting = BaseChainSetting(NETWORK_NAME=GOERLI, GENESIS_FORK_VERSION=_GOERLI_GENESIS_FORK_VERSION)
+PraterSetting = BaseChainSetting(NETWORK_NAME=PRATER, GENESIS_FORK_VERSION=_GOERLI_GENESIS_FORK_VERSION)
 # Merge Testnet (spec v1.1.9)
 KilnSetting = BaseChainSetting(NETWORK_NAME=KILN, GENESIS_FORK_VERSION=bytes.fromhex('70000069'))
 
 
 ALL_CHAINS: Dict[str, BaseChainSetting] = {
     MAINNET: MainnetSetting,
+    ROPSTEN: RopstenSetting,
+    GOERLI: GoerliSetting,
     PRATER: PraterSetting,
```

Comment:
Instead of defining `PraterSetting` separately, why don't we only define a `GoerliSetting` and just define `PRATER: GoerliSetting,` in this map? This way we clean up the duplicate settings & variables in lines 23-26 and only have to manage one `BaseChainSetting` for both.